docs/contributors/setup/set-up-git-for-ubuntu-desktop-work.md

```diff
@@ -0,0 +1,458 @@
+---
+relatedlinks: "[PackagingWithGit &#32; - &#32; Debian Wiki](https://wiki.debian.org/PackagingWithGit)"
+---
+
+(set-up-git-for-ubuntu-desktop-work)=
+# Set up Git for Ubuntu Desktop work
+
+The [Desktop Team](https://library.canonical.com/our-organisation/ubuntu-engineering/desktop) uses Git and Git Build Package (`gbp`) to maintain GNOME packages for Ubuntu. Let's set up your system so that you can contribute to GNOME on Ubuntu Desktop.
```

Comment:
The Library is internal to Canonical. These docs are public.
```suggestion
The [Desktop Team](https://discourse.ubuntu.com/c/project/desktop/8) uses Git and Git Build Package (`gbp`) to maintain GNOME packages for Ubuntu. Let's set up your system so that you can contribute to GNOME on Ubuntu Desktop.
```